src/Presets/stubs/Tailwind/styles/tailwind.config.js

```diff
@@ -64,9 +64,6 @@ module.exports = {
       },
       transparent: 'transparent',
     },
-    shadows: {
```

Comment:
This was actually doing nothing in Tailwind 1.0 either - `shadows` was renamed to `boxShadow` in v1.

Since `shadow-outline` is now deprecated, I completely removed this from the config in this case.